types/intercom-client/index.d.ts

```diff
@@ -175,3 +178,8 @@ export class Events {
     listBy(params: EventListParam): Promise<ApiResponse<CompanyList>>;
     listBy(params: EventListParam, cb: callback<ApiResponse<CompanyList>>): void;
 }
+
+export class Messages {
```

Comment:
There's no value named `Messages` in `index.js` of the original project. This doesn't quite look correct to me.

Reply:
It's inside of lib/client.js

https://developerblog.intercom.com/how-to-send-html-emails-with-the-intercom-conversations-api-748eb2784b1

Reply:
Ok, but it’s not exported from index.js, so this declaration is incorrect: 

<img width="309" alt="image" src="https://user-images.githubusercontent.com/3277153/71488821-4020aa80-27e8-11ea-9cb2-2dd171a9039a.png">

Reply:
@andrewbranch The other values such as Companies do not exist in the index of the original either, I am simply following the precedent set by the existing definition.

Reply:
Hm, you’re right. Those should be changed too.